readme.md

```diff
@@ -1,7 +1,7 @@
 # HTML form
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_html-form/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_html-form/report/html_report/)
+- [DEMO LINK](https://VladVeklenko.github.io/layout_html-form/)
+- [TEST REPORT LINK](https://VladVeklenko.github.io/layout_html-form/report/html_report/)
```

Comment:
Don't forget to add Demo and Test report links here:

<img width="1255" alt="Screenshot 2023-02-20 at 16 23 14" src="https://user-images.githubusercontent.com/104769168/220132767-2a229aad-388e-466e-82bc-06d91e7d548f.png">